scripts/benchmarks/benches/compute.py

```diff
@@ -148,14 +162,35 @@ def bin_args(self) -> list[str]:
             "--KernelExecTime=1"
         ]
 
-class SubmitKernelUR(ComputeBenchmark):
+class SubmitKernelURWithTime(ComputeBenchmark):
```

Comment:
I don't think you need to duplicate these. Simply return two values from the run() with the extra label. In fact, I think all you have to do is remove the if at line 117 and everything will just work without duplicated benchmark classes.

Reply:
I also needed to remove unit() function and set it on the result directly. Done.